### PR TITLE
Add sublime project

### PR DESCRIPTION
**Pull request by TrentHouliston**
(we're all secretly using it)

### CODE REVIEW

File: NUsight2.sublime-project [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+	"folders":
+	[
+        {
+            "folder_exclude_patterns":
+            [
+                "NUbots/build*",
+                ".vagrant"
+            ],
```
Comment (BrendanAnnable) [inline NUsight2.sublime-project]: What are these for?

**Comment by BrendanAnnable** [comment]
you monsters

File: NUsight2.sublime-project (hunk 1 above)
Comment (TrentHouliston) [reply NUsight2.sublime-project]: Oops...

File: NUsight2.sublime-project [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+	"folders":
+	[
+        {
+            "path": "."
```
Comment (BrendanAnnable) [inline NUsight2.sublime-project]: Surely this isn't still needed, is it needed? If so, can we keep the indentation consistent (2 spaces)?

Comment (TrentHouliston) [reply NUsight2.sublime-project]: path: "." is still needed for it to show up properly

**Pull request merged**